corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -20,6 +22,7 @@
 from casexml.apps.phone.restore import CachedResponse, RestoreConfig, RestoreParams, RestoreCacheSettings
 from casexml.apps.case.xml import V2, V1
 from casexml.apps.case.sharedmodels import CommCareCaseIndex
+from casexml.apps.case.const import CASE_INDEX_EXTENSION
```

Comment:
@ctsims, yes. Will merge your changes with mine, (also added a bunch of tests), then submit a PR to your repo.